eventhubs-streamanalytics-eventhubs/tools/eh-asa-perfmon/EventHandler.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Azure.Messaging.EventHubs.Consumer;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace StreamingAtScale
```

Comment:
I find it a bit concerning that this is a sample that has zero comments.  If our goal is to help walk developers through how to use the library to accomplish a given task, then it would be helpful to have some comments along the way explaining what we're doing and why.   That would help give developers context and allow them better insight for things they may want to tweak to meet their own unique application scenarios.

Reply:
comments added, details please refer to the code.